community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -216,6 +210,9 @@ public void migrate( File storeDir, File migrationDir, MigrationProgressMonitor.
                     lastTxId, lastTxInfo.checksum(), lastTxLogPosition.getLogVersion(),
                     lastTxLogPosition.getByteOffset(), progressMonitor, oldFormat, newFormat );
         }
+        // update necessary neostore records
+        LogPosition logPosition = readLastTxLogPosition( migrationDir );
+        updateOrAddNeoStoreFieldsAsPartOfMigration( migrationDir, storeDir, versionToMigrateTo, logPosition );
```

Comment:
Having this as part of migration phase rather than move face (like it was before) makes a lot more sense. Nice!

Just curious - Is this a "let's clean this up moment" or is there something else that drives this change?